textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTest.java

```diff
@@ -1,24 +1,59 @@
 package edu.uci.ics.textdb.dataflow.regexmatch;
 
-import edu.uci.ics.textdb.api.common.IPredicate;
-import edu.uci.ics.textdb.dataflow.common.SampleRegexPredicate;
-import edu.uci.ics.textdb.dataflow.source.SampleSourceOperator;
-import junit.framework.Assert;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
+import edu.uci.ics.textdb.api.common.IPredicate;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.dataflow.ISourceOperator;
+import edu.uci.ics.textdb.dataflow.common.RegexPredicate;
+import edu.uci.ics.textdb.dataflow.constants.LuceneConstants;
+import edu.uci.ics.textdb.dataflow.source.SampleDataStore;
+import edu.uci.ics.textdb.dataflow.source.ScanBasedSourceOperator;
+import edu.uci.ics.textdb.dataflow.source.TestConstants;
+
 /**
  * Created by chenli on 3/25/16.
  */
 public class RegexMatcherTest {
-
+    
+    private RegexMatcher regexMatcher;
+    
+    @Before
+    public void setUp() throws Exception{
+        SampleDataStore dataStore = new SampleDataStore();
```

Comment:
How is the "SampleDataStore" object related to the "ScanBasedSourceOperator" used later?